lib/all-badge-examples.js

```diff
@@ -781,6 +781,26 @@ const allBadgeExamples = [
           'node'
         ]
       },
+      {
+        title: 'npm bundle size (minified)',
+        previewUri: '/bundlephobia/min/react.svg',
+        keywords: [
+          'node',
+          'npm',
+          'bundle',
+          'size'
+        ]
+      },
+      {
+        title: 'npm bundle size (minified + gzip)',
+        previewUri: '/bundlephobia/gzip/react.svg',
+        keywords: [
+          'node',
+          'npm',
+          'bundle',
+          'size'
```

Comment:
You can just include `'node'` here and remove the others. Substrings of the title will be matched automatically.

Reply:
Done.